README.md

```diff
@@ -338,6 +345,25 @@ The options `-h`, `-i`, `-r`, `-f` are exclusive. The options `-s` and `-k` are
 at the LPC (where each interactive node has its own scheduler). The script can ssh to each node and run itself to modify the jobs
 on that node (because each scheduler can only be accessed for write operations from its respective node).
 
+The option `-X` will resubmit the jobs (just like `-s`), except that it will tell the job to get its input file from a specific site based on the list of sites where that file
```

Comment:
one more thing: note here (or somewhere) that this uses dasgoclient and only works for centrally produced/tracked input samples